src/app/services/baw-api/api-common.ts

```diff
@@ -46,6 +48,26 @@ export function param(x: Param) {
   return x;
 }
 
+/**
+ * Create selected timezone query string parameter (used by stringTemplate)
+ *
+ * @param delimiter QSP Delimiter
+ */
+export function timezone(delimiter: QspDelimiter) {
+  return (timezoneName: string) =>
+    `${delimiter}selected_timezone_name=${timezoneName ?? "UTC"}`;
+}
+
+/**
+ * Create authorization query string parameter (used by stringTemplate)
+ *
+ * @param delimiter QSP Delimiter
+ */
+export function authorization(delimiter: QspDelimiter) {
+  return (token?: AuthToken) =>
+    token ? `${delimiter}user_token=${token}` : "";
+}
+
```

Comment:
personally, I'd model these functions as only ever returning `key=value` and leave the delimiting up to the caller.